module-system/module-implementations/sov-evm/src/evm/primitive_types.rs

```diff
@@ -30,6 +30,20 @@ impl Default for BlockEnv {
     }
 }
 
+// BlockEnv from SealedBlock
+impl From<&SealedBlock> for BlockEnv {
+    fn from(block: &SealedBlock) -> Self {
+        Self {
+            number: block.header.number,
+            coinbase: block.header.beneficiary,
+            timestamp: block.header.timestamp,
+            prevrandao: Default::default(),
```

Comment:
If we don't support it for now let's add TODO here (preferably with an issue).

Reply:
I converted it to block.header.mix_hash, apparently we set block_env.prevrandao to mix_hash in end_slot_hook and in begin_slot_hook we set prevrandao da block hash. I don't know why we are using da root hash for these values but I converted to be consistent.